presto-tests/src/test/java/com/facebook/presto/execution/resourceGroups/db/TestQueuesDb.java

```diff
@@ -106,8 +106,7 @@ public void testRunningQuery()
         }
     }
 
-    // Disabled, please look at https://github.com/prestodb/presto/issues/16461
-    @Test(timeOut = 60_000, enabled = false)
```

Comment:
I'm a bit curious about this change. How does a disabled test fail in the first place. ?
Also the changes seem to be just removing the sleep . Does that fix the issue.?

Reply:
Ignore this, it seems to be still failling. The reason for this is: No synchronization between when the RG update happens vs a query started to run. And that leads to this situation. I am looking into the fix.